packages/grafana-flamegraph/src/FlameGraph/FlameGraph.tsx

```diff
@@ -67,118 +64,66 @@ const FlameGraph = ({
 }: Props) => {
   const styles = getStyles();
 
-  const [levels, totalProfileTicks, totalProfileTicksRight, totalViewTicks, callersCount] = useMemo(() => {
+  const [levels, levelsCallers, totalProfileTicks, totalProfileTicksRight, totalViewTicks] = useMemo(() => {
     let levels = data.getLevels();
     let totalProfileTicks = levels.length ? levels[0][0].value : 0;
     let totalProfileTicksRight = levels.length ? levels[0][0].valueRight : undefined;
-    let callersCount = 0;
     let totalViewTicks = totalProfileTicks;
+    let levelsCallers = undefined;
 
     if (sandwichItem) {
       const [callers, callees] = data.getSandwichLevels(sandwichItem);
-      levels = [...callers, [], ...callees];
-      // We need this separate as in case of diff profile we to compute diff colors based on the original ticks.
+      levels = callees;
+      levelsCallers = callers;
```

Comment:
Before we had both callers and callees levels in the same array with one blank row. This worked neatly with the old rendering by row but now we need separate graphs and we also render them separately in two components.